### ISSUE TITLE
Sample code Simulate mode  always mute?

### ISSUE THREAD

**Issue by rex0521** [mapbox/mapbox-navigation-ios#854]
hello
i build sample code and startNavigation
but Navigation process always mute.
system show "The directions request was made without `includesVoiceInstructions` enabled. This will prevent users from getting voice instructions. It's recommended to make your directions request via `NavigationRouteOptions()`."

thx

**Comment by frederoni** [comment]
What version of the SDK are you using? What's your iPhone Language set to?
When you issue the request to MapboxDirections.swift, do you use `NavigationRouteOptions` instead of `RouteOptions`?

**Comment by rex0521** [comment]
What version of the SDK are you using? 
-->https://www.mapbox.com/ios-sdk/Mapbox-iOS-SDK.json" ~> 3.6
What's your iPhone Language set to?
-->chinese,Traditional(Taiwan)
When you issue the request to MapboxDirections.swift, do you use NavigationRouteOptions instead of RouteOptions?
-->i download mapbox-navigation-ios-master project and build Objective-C sample code
     ;obj-c sample code use MBRouteOptions MBDirections MBRoute MGLPolyline 
     MBSimulatedLocationManager startNavigation.

thx

**Comment by frederoni** [comment]
Just noticed the Objective-C example is using the wrong options.
Can you try changing this line:
```objc
MBRouteOptions *options = [[MBRouteOptions alloc] initWithWaypoints:waypoints profileIdentifier:MBDirectionsProfileIdentifierAutomobileAvoidingTraffic];
```
to:
```objc
MBNavigationRouteOptions *options = [[MBNavigationRouteOptions alloc] initWithWaypoints:waypoints profileIdentifier:MBDirectionsProfileIdentifierAutomobileAvoidingTraffic];
```

**Comment by frederoni** [comment]
Fixed in #857

**Comment by rex0521** [comment]
hi frederoni 
thank you 
it is working!!!!!!
